bnr-utils/nz_azConnector/nz_azConnector.go

```diff
@@ -301,9 +312,72 @@ func (cn *Conn) downloadBkp(outdir string, uniqueid string, blobpath string, str
     close(work)
     <- done
     log.Println("Total files downloaded:", filesdownloaded)
+    updateLocation(arrayLoc,outdir)
+    updateContents(arrayContents)
     return err
 }
 
+func updateLocation(arrLoc []string,outdir string){
+    for _,locFile := range arrLoc{
+        input, err := ioutil.ReadFile(locFile)
+        if err != nil {
+            log.Println("Unable to Override Contents and Location as download dir not Present.")
+            log.Fatalln(err)
+        }
+        lines := strings.Split(string(input), "\n")
+        if (len(lines) == 2 && !strings.HasSuffix(lines[len(lines) -2] , outdir)) {
+            f, err := os.OpenFile(locFile,
+                os.O_APPEND|os.O_CREATE|os.O_WRONLY, 0644)
+            if err != nil {
+                log.Println("Unable to Override Location as Unable to open file.")
+                log.Fatalln(err)
+            }
+            defer f.Close()
+            textAppend := "1,1,1," + outdir + "\n"
+            if _, err := f.WriteString(textAppend); err != nil {
+                log.Println("Unable to Override Location as Write to File Failed.")
+                log.Fatalln(err)
+            log.Println("Location.txt override for cloud  backup")

```

Comment:
Remove this log.

Reply:
Removed :
log.Println("Location.txt override for cloud  backup") log 
and combined ,
log.Println("Unable to Override Location as Write to File Failed.")
log.Fatalln(err)